analyzers/tests/SonarAnalyzer.UnitTest/TestCases/EqualityOnFloatingPoint.cs

```diff
@@ -2,86 +2,132 @@
 using System.Collections.Generic;
 using System.Runtime.Versioning;
 
-namespace Tests.Diagnostics
+public class EqualityOnFloatingPoint
 {
-    public class EqualityOnFloatingPoint
+    void Test(float f, double d1, double d2)
     {
-        void test(float f, double d1, double d2)
+        dynamic din = null;
```

Comment:
Old code: It's weird that DYNamic is called DIN

Reply:
Fixed... and made it somewhat useful too.